qa/suites/upgrade/octopus-x/stress-split/1-ceph-install/nautilus.yaml

```diff
@@ -0,0 +1,31 @@
+meta:
+- desc: install ceph/nautilus latest
+tasks:
+- install:
+    exclude_packages:
+      - ceph-mgr-cephadm
+      - cephadm
+      - libcephfs-dev
+    branch: nautilus
+- print: "**** done install nautilus"
+- ceph:
+    conf:
+      global:
+        bluestore_warn_on_legacy_statfs: false
+        bluestore warn on no per pool omap: false
+        mon pg warn min per osd: 0
+    log-whitelist:
+      - Not found or unloadable
+- exec:
+    osd.0:
+      - ceph osd require-osd-release nautilus
+      - ceph osd set-require-min-compat-client nautilus
+- print: "**** done ceph"
+- rgw:
+   - client.0
+- print: "**** done => started rgw client.0"
+overrides:
+  ceph:
+    conf:
+      mon:
+        mon warn on osd down out interval zero: false
```

Comment:
question is: do we want to automatically adopt the nautilus cluster to cephadm and then upgrade to pacific?

Reply:
I think we should add that to the nautilus-x suite for pacific. this octopus-x suite shouldn't have nautilus

Reply:
We have not changed anything in `splits` yet
Noted.